src/modules/src/stabilizer.c

```diff
@@ -256,6 +259,12 @@ static void stabilizerTask(void* param)
       stateEstimator(&state, tick);
       compressState();
 
+      if (RATE_DO_EXECUTE(RATE_HL_COMMANDER, tick)) {
```

Comment:
Currently I'm wondering if this is the right place to handle the planner and the execution rate of the high level commander. The position and attitude rate are handled within the controllers so perhaps this part should be in commander.c ? Just thinking out loud how to improve the consistency of the code

Reply:
OK, but wouldn't it go in crtp_commander_high_level.c?

In the long term I think we might want to move the rate-skipping code in stabilizer.c, because as we make more modules bind-able it would not make sense to include the rate limiting within the bound "library" part. But that is orthogonal to this PR.

Reply:
moved in 65f3a41

Reply:
yeah indeed, let's keep it in mind for future discussions!